### ISSUE TITLE
TF Provider fails with the message

### ISSUE THREAD

**Issue by surajsub** [IBM-Cloud/terraform-provider-ibm#614]
### Terraform Version
0.11.11
0.17.0
### Affected Resource(s)
Please list the resources as a list, for example:
- ibm_network_public_ip

### Terraform Configuration Files
```hcl
## Template to get a public ip from SL


provider ibm {
version="0.17.0"
} 


variable "routes_to"
{
description="IP Address to route to"
}

variable "notes"
{
description="Information about the Public IP

}


resource "ibm_network_public_ip" "publicIp" {
    routes_to = "${var.routes_to}"
    notes     = "${var.notes}"
}


```

### Debug Output
Please provider a link to a GitHub Gist containing the complete debug output: https://www.terraform.io/docs/internals/debugging.html. Please do NOT paste the debug output in the issue; just paste a link to the Gist.

* ibm_network_public_ip.publicIp: Error during creation of network public ip: couldn't find resource (21 retries)
2019/06/24 17:04:28 [ERROR] root: eval: *terraform.EvalSequence, err: 1 error(s) occurred:

* ibm_network_public_ip.publicIp: Error during creation of network public ip: couldn't find resource (21 retries)

### Panic Output
No Panic 
### Expected Behavior
What should have happened?

The public_ip should have been created

### Actual Behavior
What actually happened?

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `terraform apply`

**Comment by surajsub** [comment]
@stevestrutt  - @hkantare  Can you shed some light on this please ?

**Comment by hkantare** [comment]
We are working on this..

**Comment by hkantare** [comment]
Fix is available as part of master branch https://github.com/IBM-Cloud/terraform-provider-ibm

**Comment by hkantare** [comment]
Available as part of v0.17.2 release..
https://github.com/IBM-Cloud/terraform-provider-ibm/releases/tag/v0.17.2

**Comment by hkantare** [comment]
closing the issue
